pipelines/rj_smtr/tasks.py

```diff
@@ -760,6 +760,9 @@ def create_request_params(
     elif dataset_id == constants.GTFS_DATASET_ID.value:
```

Comment:
Da pra fazer: `dataset_id in [constants.GTFS_DATASET_ID.value, constants.GTFS_DATASET_ID_NEW.value]` para não ter que repetir o mesmo if pro antigo e pro novo